solution/ui/regulations/utilities/api.js

```diff
@@ -390,21 +390,9 @@ const getLastUpdatedDates = async (apiUrl, titleArr = ["42"]) => {
         titleArr.map((title) => httpApiGet(`title/${title}/parts`))
     );
 
-    const combinedResults = results.flat(1).reduce(
-        (accumulator, current) => ({
-            ...accumulator,
-            [current.name]: current,
-        }),
-        {}
-    );
-
-    // remove artifact added by front end caching
-    delete combinedResults.expiration_date;
-
-    return Object.fromEntries(
-        Object.entries(combinedResults).map((arr) => [arr[1].name, arr[1].date])
-    );
+    return createLastUpdatedDates(results);
```

Comment:
break this logic out into a utility function so it can be used in some unit tests